study.md

```diff
@@ -27,17 +27,20 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+The DOM is not source and
+it's also not a rendered page.
+
+
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+Yes and no if you leave empty tags sometimes the browser fills in the blanks.
 ```
```

Comment:
Your answer 'yes and no' is correct!!! What we see in Dev Tools is a visual representation of the DOM. Sometimes the DOM will correct broken HTML, but we should NEVER be dependent on this!